starter-code/server.js

```diff
@@ -75,24 +79,28 @@ app.put('/articles/:id', function(request, response) {
   // an author_id property, so we can reference it from the request.body.
   // TODO: Add the required values from the request as data for the SQL query to interpolate
   client.query(
-    ``,
-    []
+    `UPDATE authors
+    SET author = $1, "authorUrl" = $2
+    WHERE author_id = $3`,
+    [request.body.author, request.body.authorUrl, request.body.author_id]
   )
   .then(function() {
     // TODO: Write a SQL query to update an article record. Keep in mind that article records
     // now have an author_id, in addition to title, category, publishedOn, and body.
     // TODO: Add the required values from the request as data for the SQL query to interpolate
     client.query(
-      ``,
-      []
-    )
+      `UPDATE articles
+      SET title = $2, category = $3, "publishedOn" = $4, body = $5
+      WHERE author_id = $1;`,
+      [request.body.author_id, request.body.title, request.body.category, request.body.publishedOn, request.body.body]
```

Comment:
`app.put('/articles/:id', function(request, response) {`
so here we have the `id` (`request.params.id`)